buildsystem/modules/FindPython.cmake

```diff
@@ -182,7 +182,7 @@ foreach(PYTHON ${PYTHON_INTERPRETERS})
 		"${CMAKE_BINARY_DIR}"
 		SOURCES "${CMAKE_CURRENT_LIST_DIR}/FindPython_test.cpp"
 		LINK_LIBRARIES "${PYTHON_LIBRARY_NAME}"
-		CMAKE_FLAGS "-DINCLUDE_DIRECTORIES=${PYTHON_INCLUDE_DIR}" "-DLINK_DIRECTORIES=${PYTHON_LIBRARY_DIR}"
+		CMAKE_FLAGS "-DINCLUDE_DIRECTORIES=${PYTHON_INCLUDE_DIR}" "-DLINK_DIRECTORIES=${PYTHON_LIBRARY_DIR}" "-DCMAKE_CXX_STANDARD=14"
```

Comment:
Nope, that is happening in a dynamically generated CMake project. `CXX_STANDARD` and some other stuff do not get handed through. This is already fixed upstream, but those changes aren't on my system yet.